rsopt/pkcli/pack.py

```diff
@@ -86,21 +96,27 @@ def _create_tar(name, file_list):
 
     return tarname
 
+
 def configuration(config, ignore=None, add=None):
```

Comment:
What is ignore and add? No doctring or type hint

Reply:
I knew I had already written this. This branch is kind of old I guess.
0191fd19e5bcfd8c488cdd4fbeeede5a3d0ef435